app/shared/sidebar/_sidebar.scss

```diff
@@ -0,0 +1,5 @@
+.app-Sidebar {
+  &__content{
+    right: 0 !important;
+  }
```

Comment:
This forces the content to stay behind the sidebar, otherwise react-sidebar calculates the sidebar width and resizes the content.